frontend/app/src/composables/update-message.ts

```diff
@@ -0,0 +1,34 @@
+export const useUpdateMessage = createSharedComposable(() => {
+  const lastUsedVersion = useLocalStorage<string>('rotki.last_version', null);
+  const showReleaseNotes = ref(false);
+
+  const { appVersion } = storeToRefs(useMainStore());
+
+  const link = computed(
+    () => `https://github.com/rotki/rotki/releases/tag/${get(appVersion)}`
+  );
+
+  const setShowNotes = (appVersion: string, lastUsed: string): void => {
+    if (!appVersion || appVersion.includes('dev')) {
```

Comment:
will this not prevent it from showing on dev...

Reply:
The idea here is that the message should only show before the first login on a new production version.

Reply:
It doesn't make sense to show it on dev since the link will not work either way.